.storybook/main.ts

```diff
@@ -19,9 +23,9 @@ module.exports = {
     ],
 
     addons: [
-        getAbsolutePath("@storybook/addon-links"),
-        getAbsolutePath("@storybook/addon-essentials"),
-        getAbsolutePath("@storybook/addon-a11y"),
+        "@storybook/addon-links",
+        "@storybook/addon-essentials",
+        "@storybook/addon-a11y",
```

Comment:
I created `getAbsolutePath()` in the original PR where we upgraded to Storybook v7, but I can't recall why and removing it works just fine!  😖